templates/default/interfaces.erb

```diff
@@ -1,7 +1,7 @@
 <% if @auto -%>
 auto <%= @device %>
 <% end %>
-iface <%= @device %> <%= @family %> <%= @type %>
+iface <%= @device %> <%= @family %> <%= @method %>
```

Comment:
I don't understand this change. You exposed a `type` attribute that gets propagated into the template but in the template you stop using it, instead using `method` which is _not_ set anywhere. Was this change a mistake?

Reply:
@Fuuzetsu I think you’re right! Change landed in 22520e9f017fdf7f2982a213668ff265f85f3518.

I’ll look through things more closely & change things if, indeed, it really is a mistake.

Reply:
Yep! Fixing in #43